Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN --mount=type=secret,id=NEXT_PUBLIC_HCAPTCHA_SITEKEY \
     export NEXT_PUBLIC_SENTRY_DSN=$(cat /run/secrets/NEXT_PUBLIC_SENTRY_DSN) && \
     export SENTRY_ORG="prezly" && \
     export SENTRY_PROJECT="theme-nextjs-bea" && \
+    export PREZLY_THEME_UUID="73015107-ac86-418b-9120-4ffa439d5c0f" && \
```

Comment:
Have you checked with Camil about this?

Reply:
@oleg-semyonov Do you need this during the build only? This will be active only during this phase. If you need it at runtime, we need to set it in k8s deployment.

Reply:
@oleg-semyonov Done in https://github.com/prezly/eks/pull/28